x-pack/plugins/security_solution/public/cloud_security_posture/components/vulnerabilities/vulnerabilities_preview.tsx

```diff
@@ -103,49 +95,22 @@ export const VulnerabilitiesPreview = ({
 
   const { euiTheme } = useEuiTheme();
 
-  const { data: dataMisconfiguration } = useMisconfigurationPreview({
-    query: buildEntityFlyoutPreviewQuery('host.name', name),
-    sort: [],
-    enabled: true,
-    pageSize: 1,
-  });
-
-  const passedFindings = dataMisconfiguration?.count.passed || 0;
-  const failedFindings = dataMisconfiguration?.count.failed || 0;
+  const { hasMisconfigurationFindings } = useHasMisconfigurations('host.name', name);
```

Comment:
let's pass the fieldName instead of `host.name` directly. Even though vuln preview don't show up on user flyout, this code opens up the door for bugs if we ever have the vuln for users